lua/dap.lua

```diff
@@ -1087,31 +1087,32 @@ function M.step_into()
         if not target or not target.id then
           print('No target selected. No stepping.')
         else
-          session:_step('stepIn', { target_id = target.id })
+          options.target_id = target.id
```

Comment:
Same here, might cause an error if `options` is `nil`. Could also add a `options = options or {}` at the start of the method to avoid that.


Btw. should we rename `options` to `opts` ? For consistency as that's already used in other places. (I went with `opts` instead of the more verbose variant because it's also commonly used in the neovim code base)